src/Altinn.AccessManagement/Controllers/LookupController.cs

```diff
@@ -59,7 +59,15 @@ public async Task<ActionResult<PartyExternal>> GetOrganisation(string orgNummer)
                 }
 
                 Party party = await _register.GetOrganisation(orgNummer);
-                return _mapper.Map<PartyExternal>(party);
+
+                if (party == null)
+                {
+                    return new ObjectResult(ProblemDetailsFactory.CreateValidationProblemDetails(HttpContext, ModelState, 400));
```

Comment:
Without adding an error to the ModelState the object result model won't really have much value.
But the entire Lookup controller will be moved to the BFF anyway so it will all depend on how errors are to be handled between the BFF and frontend. So I guess this is ok for now. At least we're no longer responding will 200 and null object